tests/inventory/pipelines/base_pipeline_test.py

```diff
@@ -87,11 +88,11 @@ def test_get_loaded_count(self):
     def test_error_is_handled_in_get_loaded_count(self):
         """Test error from get_loaded_count is handled."""
 
-        self.pipeline.logger = mock.create_autospec(
-            LogUtil).setup_logging('foo')
+        base_pipeline.LOGGER = mock.create_autospec(
```

Comment:
@blueandgold FYI I tweaked your pipeline unit tests to reference the [MODULE_NAME].LOGGER property and made the pipeline.logger property a global "LOGGER" variable to be consistent across the codebase. Let me know if this looks ok; I'm not totally sure what's the best way here.